openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -100,7 +100,7 @@ def test_queries(self):
         course_home_url(self.course)
 
         # Fetch the view and verify the query counts
-        with self.assertNumQueries(43):
+        with self.assertNumQueries(44):
```

Comment:
@UmanShahzad Did you enjoy this test having super-intimate knowledge of what happens under the hood? :) No action needed, obviously.